tck/logs/README.adoc

```diff
@@ -49,7 +49,7 @@ To enable the tests in your project you need to add the following dependency to
 
 ----
 
-If you want to run the logs tests, you can specify all tests in the `tck-suite.xml`. E.g. 
+If you want to run the logs tests, you can specify the application logging tests in the `tck-suite.xml`. E.g. 
```

Comment:
does there need to be something said here about not configuring the `otel.sdk.disabled=false`/`OTEL_SDK_DISABLED=FALSE` as a system property or environment variable?

Reply:
change to: (to avoid making it sound optional)
To run the logs tests, specify the application logging tests in the `tck-suite.xml`. E.g.